src/test/java/com/kodilla/ecommercee/entity/ProductEntityTest.java

```diff
@@ -1,7 +1,113 @@
 package com.kodilla.ecommercee.entity;
 
+import com.kodilla.ecommercee.repository.GroupRepository;
+import com.kodilla.ecommercee.repository.ProductRepository;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.math.BigDecimal;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
 import static org.junit.Assert.*;
 
+@RunWith(SpringRunner.class)
+@SpringBootTest
 public class ProductEntityTest {
+    private ProductGroupEntity group;
+    private ProductEntity productEntity;
+    @Autowired
+    private ProductRepository productRepository;
+
+    @Autowired
+    private GroupRepository groupRepository;
+
+    @Before
```

Comment:
Uważam że czytelniej było by zrezygnować z @Before i @After bo kiedy czytam test to muszę skakać oczami od pozycji do pozycji zamiast wygodnie przeczytac od góry do dołu. Wydaje mi się że lepiej jest kiedy test zaczyna i kończy się w obrębie samego siebie.